### PR TITLE
ensure 4.9 openshift clusters can be managed by hypershift

### PR DESCRIPTION
**Pull request by relyt0925**
IBM has a large estate of production 4.9_openshift hypershift clusters. 4.9 cannot be deemed unsupported until a strategy is worked out and communicated to customers

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: support/supportedversion/version.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ var MinSupportedVersion = semver.MustParse(subtractMinor(&LatestSupportedVersion
 
 // SupportedPreviousMinorVersions is the number of minor versions prior to current
 // version that are supported.
-const SupportedPreviousMinorVersions = 2
+const SupportedPreviousMinorVersions = 3
```
Comment (enxebre) [inline support/supportedversion/version.go:18]: Our generic generic policy is N-2. Can we make this IBM specific in the IsValidReleaseVersion logic?

Comment (relyt0925) [reply support/supportedversion/version.go:18]: We can!

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**